install-scripts/mac/skm-install-mac.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+GIT_SKM_REPO=https://github.com/splashkit/skm.git
+MARK_CHECK="\xE2\x9C\x94"
+MARK_CROSS="\xE2\x9C\x95"
+
+HOME_PATH=~
+INSTALL_PATH="${HOME_PATH}/.splashkit"
+
+function has_git() {
+	git --help 2>&1 > /dev/null
+	return $?
+}
+
+function install_git() {
+	echo "macOS developer tools are currently not installed which are required for installation skm."
+	read -p "Would you like to install them now? " -n 1 -r
+	echo ""
+	if [[ $REPLY =~ [Yy]$ ]]
+	then
+		xcode-select --install
+	else
+    	echo "macOS developer tools not installed, please run: \"xcode-select --install\" in the terminal and then reinstall."
+		exit 1
+	fi
+}
+
+echo "Checking for macOS developer tools..."
+has_git
+if [ $? -ne 0 ]; then
+	install_git
+	has_git
+	if [ $? -ne 0 ]; then
```

Comment:
**suggestion:** Similar to the above,

```suggestion
	if has_git; then
```

Reply:
Agreed on above, but disagree on this. The only branching that needs to occur is when `has_git` returns `!= 0`. I think explicitly stating this is more obvious for reading.

Reply:
Ah gotcha,  could we do `if ! has_git; then`?

Reply:
Fixed in 4a475c1a8287a1d7c822ef0ae28cd672d1c77f3a